src/citrine/_session.py

```diff
@@ -37,7 +37,8 @@ def __init__(self,
                  port: Optional[str] = None):
         super().__init__()
         self.scheme: str = scheme
-        self.authority = ':'.join([host] if host else [] + [port] if port else [])
+        # self.authority = ':'.join([host] if host else [] + [port] if port else [])
+        self.authority = ':'.join([host or '', port or ''])
```

Comment:
```suggestion
        self.authority = ':'.join(([host] if host else []) + [port] if port else [])
```
The actual fix; poor testing on my part, sorry.